internal/distro/rhel86/distro.go

```diff
@@ -1757,24 +1808,66 @@ func newDistro(distroName string) distro.Distro {
 	ociImgType := qcow2ImgType
 	ociImgType.name = "oci"
 
-	x86_64.addImageTypes(qcow2ImgType, vhdImgType, vmdkImgType, openstackImgType, amiImgTypeX86_64, tarImgType, imageInstaller, edgeCommitImgType, edgeInstallerImgType, edgeOCIImgType, edgeRawImgType, edgeSimplifiedInstallerImgType, ociImgType, gceImgType)
-	aarch64.addImageTypes(qcow2ImgType, openstackImgType, amiImgTypeAarch64, tarImgType, imageInstaller, edgeCommitImgType, edgeInstallerImgType, edgeOCIImgType, edgeRawImgType, edgeSimplifiedInstallerImgType)
-	ppc64le.addImageTypes(qcow2ImgType, tarImgType)
-	s390x.addImageTypes(qcow2ImgType, tarImgType)
+	x86_64.addImageTypes(
+		amiImgTypeX86_64,
+		edgeCommitImgType,
+		edgeInstallerImgType,
+		edgeOCIImgType,
+		gceImgType,
+		imageInstaller,
+		ociImgType,
+		openstackImgType,
+		qcow2ImgType,
+		tarImgType,
+		vhdImgType,
+		vmdkImgType,
+	)
+
+	aarch64.addImageTypes(
+		amiImgTypeAarch64,
+		edgeCommitImgType,
+		edgeInstallerImgType,
+		edgeOCIImgType,
+		imageInstaller,
+		openstackImgType,
+		qcow2ImgType,
+		tarImgType,
+	)
+
+	ppc64le.addImageTypes(
+		qcow2ImgType,
+		tarImgType,
+	)
+
+	s390x.addImageTypes(
+		qcow2ImgType,
+		tarImgType,
+	)
 
 	if rd.isRHEL() {
 		// add azure to RHEL distro only
 		x86_64.addImageTypes(azureRhuiImgType)
 
 		// add ec2 image types to RHEL distro only
-		x86_64.addImageTypes(ec2ImgTypeX86_64, ec2HaImgTypeX86_64, ec2SapImgTypeX86_64)
+		x86_64.addImageTypes(ec2ImgTypeX86_64, ec2HaImgTypeX86_64)
+		if !versionLessThan(rd.osVersion, "8.6") {
+			x86_64.addImageTypes(ec2SapImgTypeX86_64)
+
+			// edge simplified installer is only available on 8.6 and later on RHEL
+			x86_64.addImageTypes(edgeSimplifiedInstallerImgType, edgeRawImgType)
+			aarch64.addImageTypes(edgeSimplifiedInstallerImgType, edgeRawImgType)
+		}
```

Comment:
What is the risk of not making this distinction for older RHEL releases? It is not like we would be introducing these image types to the RHEL-8.5 which is already released. Also one can not build 8.5 in the service, because AFAIK it supports only the latest RHEL.

So if these images can be built with RHEL-8.5 repositories, I would be for supporting them there just to make things simpler.

Reply:
The images can't be built on 8.5 anymore.  The FDO packages we added in 8.6 aren't available in 8.5.  It's simple enough to conditionally add packages, but there's LUKS and FDO things all over the pipelines that we'd have to conditionally toggle.  Supporting them in 8.5 would add a lot of mess to the pipelines for no benefit since, afaict, no one would want to build these for 8.5.